src/control/controlbehavior.h

```diff
@@ -30,6 +30,15 @@ class ControlNumericBehavior {
             MidiOpCode o, double dParam, ControlDoublePrivate* pControl);
 };
 
+// ControlSelectBehavior passes the midi value directly to the internal parameter value.  It's
+// useful for selector knobs that pass +1 in one direction and -1 in the other.
+class ControlSelectBehavior : public ControlNumericBehavior {
```

Comment:
I am not sure about the name. A selector is for me a latching rotary with steps. 
What you have here is an encoder rotary that encodes the rate of turn. 

We currently implement a selector as a toggle button, toggling though the states. It would be odd to a require here a new control if one likes to bind a latching selecting rotary. So I would prefer to handle a selector also with a ControlPushButton.  That is out of scope of this PR. 

So I think this can become safely ControlEncoderBehaviour.

Reply:
I am happy to change the name.  "Encoder" is much better